1/contrib/s2i/run

```diff
@@ -86,7 +104,7 @@ rm -rf /tmp/war
 
 # if `docker run` first argument start with `--` the user is passing jenkins launcher arguments
 if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
-   exec java $JAVA_OPTS -Dfile.encoding=UTF8 -jar /usr/lib/jenkins/jenkins.war $JENKINS_OPTS "$@"
+   exec java $JAVA_MAX_HEAP_PARAM $JAVA_OPTS -Dfile.encoding=UTF8 -jar /usr/lib/jenkins/jenkins.war $JENKINS_OPTS "$@"
```

Comment:
if JAVA_OPTS contains a -Xmx argument, it will take precedence over the JAVA_MAX_HEAP_PARAM value, right?  Just want to make sure users have a way to explicitly override the max heap setting if they want to.